scout/dashboard.md

```diff
@@ -17,6 +17,19 @@ The Docker Scout Dashboard helps you share the analysis of images in an organiza
 
 The **Overview** tab shows the total number of vulnerabilities across all your Scout-enabled repositories, over time. This calculation takes the most recent image in each repository to avoid including old irrelevant images.
 
+## Repository settings
+
+Enable Docker Scout analysis on repositories in the current organization by selecting your profile
+picture and then the **Repository settings** menu item.
+
+Select the check boxes for the repositories on which you want to enable Docker Scout analysis and
```

Comment:
```suggestion
Select the checkboxes for the repositories on which you want to enable Docker Scout analysis and
```

Reply:
Ahh, that didn't trip the style checker because of the pluralisation I think, will fix that.